<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>propertiesDialog</class>
 <widget class="QDialog" name="propertiesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>590</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Цветовая схема</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/qwin_otter/Otter</normaloff>:/qwin_otter/Otter</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="Layout_cell">
     <item>
      <widget class="QLabel" name="label_cell_x">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>delt X:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="param_ed" name="lineCellX">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>10</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_cell_y">
       <property name="text">
        <string>delt Y:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="param_ed" name="lineCellY">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>10</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_thicks">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_thick_osc">
       <item>
        <widget class="QLabel" name="label_thick_sel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Thick select</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="param_ed" name="ed_thick_sel">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>2</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_thick_strob">
       <item>
        <widget class="QLabel" name="label_thick_rule">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Thick rule</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="param_ed" name="ed_thick_rule">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>2</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="Layout_col_background">
     <item>
      <widget class="QLabel" name="lab_col_background">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Background color</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButtonColored" name="Button_col_background">
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="Layout_col_ocs">
     <item>
      <widget class="QLabel" name="label_sel">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Select color</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButtonColored" name="Button_col_sel">
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="Layout_col_cell">
     <item>
      <widget class="QLabel" name="label_col_rule">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Ruler color</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButtonColored" name="Button_col_rule">
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="Layout_col_cell1">
     <item>
      <widget class="QLabel" name="label_col_rule1">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Ruler color</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButtonColored" name="Button_col_rule1">
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="Layout_col_strob">
     <item>
      <widget class="QLabel" name="label_col_circle">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Circle color</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButtonColored" name="Button_col_circle">
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_circ">
     <item>
      <widget class="QLabel" name="label_c_thick">
       <property name="text">
        <string>Circle thick:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="param_ed" name="ed_circle_thick">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_circ_rad">
       <property name="text">
        <string>Circle rad:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="param_ed" name="ed_circle_rad">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="Layout_col_text">
     <item>
      <widget class="QLabel" name="label_col_crs_2">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Сrossline color:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButtonColored" name="Button_col_crs">
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="Layout_col_sepLine">
     <item>
      <widget class="QLabel" name="label_col_crs">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Сrossline thick:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="param_ed" name="ed_cross_thick">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="Layout_col_marker">
     <item>
      <widget class="QLabel" name="label_col_grid">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Grid</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButtonColored" name="Button_col_grid">
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="Layout_col_bg_lib">
     <item>
      <widget class="QLabel" name="label_col_bg_lib">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Lib BG color</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButtonColored" name="Button_col_lib_bg">
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="Layout_col_lib_grid">
     <item>
      <widget class="QLabel" name="label_col_lib_grid">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Lib grid color</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButtonColored" name="Button_col_lib_grid"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_circ1">
     <item>
      <widget class="QLabel" name="label_lib_x">
       <property name="text">
        <string>Lib grid X:</string>
       </property>
       <property name="text1" stdset="0">
        <string>Circle tick1:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="param_ed" name="ed_lib_gr_x">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize1" stdset="0">
        <size>
         <width>50</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_circ_rad1">
       <property name="text">
        <string>Lib grid Y:</string>
       </property>
       <property name="text1" stdset="0">
        <string>Circle rad1:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="param_ed" name="ed_lib_gr_y">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize1" stdset="0">
        <size>
         <width>50</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="Layout_col_no_ak_2">
     <item>
      <widget class="QLabel" name="label_col_no_ak">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>No AK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButtonColored" name="Button_col_no_ak"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="Layout_buttons">
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QPushButtonColored</class>
   <extends>QPushButton</extends>
   <header location="global">QPushButtonColored.h</header>
  </customwidget>
  <customwidget>
   <class>param_ed</class>
   <extends>QLineEdit</extends>
   <header location="global">param_ed.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="win_snail.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>propertiesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>78</x>
     <y>476</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>propertiesDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>221</x>
     <y>476</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
